index.js

```diff
@@ -47,16 +42,15 @@ module.exports = function buildAirbnbPreset(context, options) {
         ignoreFilenames: ['node_modules']
       }, options.removePropTypes)] : null,
 
-      options && options.modules === false ? null : modules,
-      options && options.modules === false ? null : ['babel-plugin-transform-strict-mode', { strictMode: true }],
```

Comment:
`preset-env` is now `"auto"` by default for options. Might be good to preserve that behavior by just passing `options.modules` through to `preset-env` instead of reimplementing it.